codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java

```diff
@@ -97,6 +100,14 @@ public JacksonSerializer() {
                 }
             }
         }
+
+    }
+
+    //注册泛化接口对应的实际实现类型
+    public static void registerGenericService(String serviceName, String className) {
```

Comment:
Now, this method is called by test case. When does it call in real application using RPC?

Reply:
This is a good question!   In fact, the method is not called by test case, it can be used in the scenario that someone want to publish a generic interface and use json to serialize/deserialize.
What we need to discuss is how to implement this better.  Except the above way, we also can add the field in `ProviderConfig`, any suggestion?

Reply:
When we are doing  `ProviderCofnig.export()`, we can do it.